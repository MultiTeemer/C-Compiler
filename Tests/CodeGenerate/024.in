struct point { 
	int x, y;
	struct {
		int ro, fi;
	} z;	
} a;

int main()
{
	a.x = 3;
	a.y -= 15;
	a.z.ro = a.x - a.y;
	a.z.fi += a.z.ro - a.y;
	printf("%d %d %d %d", a.x, a.y, a.z.ro, a.z.fi);
}