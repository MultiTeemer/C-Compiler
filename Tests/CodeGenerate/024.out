3 -15 18 33